src/windows/functions.cpp

```diff
@@ -303,54 +314,80 @@ MOUNTUTILS_RESULT EjectDriveLetter(TCHAR driveLetter) {
   HANDLE volumeHandle = CreateVolumeHandleFromDriveLetter(driveLetter,
                                                           volumeFlags);
 
+  MountUtilsLog("Creating volume handle");
+
   if (volumeHandle == INVALID_HANDLE_VALUE) {
+    MountUtilsLog("Couldn't create volume handle");
     return MOUNTUTILS_ERROR_INVALID_DRIVE;
   }
 
   // Don't proceed if the volume is not mounted
   if (!IsVolumeMounted(volumeHandle)) {
+    MountUtilsLog("Volume is not mounted");
+
     if (!CloseHandle(volumeHandle)) {
+      MountUtilsLog("Couldn't close volume handle");
       return MOUNTUTILS_ERROR_GENERAL;
     }
 
     return MOUNTUTILS_SUCCESS;
   }
 
   if (IsDriveFixed(driveLetter)) {
+    MountUtilsLog("Drive is a fixed");
```

Comment:
Sounds like this should either be "Drive is a fixed drive" or "Drive is fixed" ?